src/dotnet/commands/dotnet-test/LocalizableStrings.cs

```diff
@@ -60,5 +60,10 @@ internal class LocalizableStrings
 
         public const string RunSettingsArgsHelpText = @"Any extra command-line runsettings arguments that should be passed to vstest. See 'dotnet vstest --help' for available options.
                                         Example: -- RunConfiguration.ResultsDirectory=""C:\users\user\desktop\Results Directory"" MSTest.DeploymentEnabled=false";
+
+        public const string CmdResultsDirectoryDescription = @"The test results directory will be created in the specified path if it does not exist.
```

Comment:
Update to "The directory where tests results will be placed. Specified directory will be created if it does not exist."